src/components/badgeable/__tests__/Badgeable-test.js

```diff
@@ -10,6 +10,18 @@ describe('components/badgeable/Badgeable', () => {
         expect(wrapper).toMatchSnapshot();
     });
 
+    test('should pass down props to children when it is an element', () => {
+        const children = <div id="render">Some text to render</div>;
+        const wrapper = shallow(<Badgeable foo="bar">{children}</Badgeable>);
```

Comment:
To make this more deliberate, let's `shallow` and include the output for `children` in the wrapper directly.

```
shallow(
  <Badgeable foo="bar">
    <div id="render">etc.</div>
  </Badgeable>
);

Let's also add a test to define what should happen if `children` is more than one Element.